src/engine/EventManager.cxx

```diff
@@ -292,17 +311,12 @@ void EventManager::checkEvents(SDL_Event &event, Engine &engine)
         }
         else if (!tileTypeEditMode.empty())
         {
-          if (m_highlightedNodes.size() == 0)
+          std::vector<Point> vec = {m_clickDownCoords};
```

Comment:
Why are you making a copy of m_clickDownCoords here? If using Iterators this would be simpler

Reply:
I'll try to fix that also in the next PR